src/css/profile/mobile/common/popup.less

```diff
@@ -163,6 +187,28 @@
 		flex-grow: 0;
 		width: 100%;
 
+		&::before {
```

Comment:
In header you used `::after` but in footer you use `::before`. Is possible to use the same type of pseudo element?

Reply:
A divider should be created at the bottom of the header.
That's why I use ::after.
The divider is also created in the upper part of the footer, so ::before should be used.
This is the intended result.